pkgs/applications/misc/collision/gi-crystal/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, crystal
+, libadwaita
+, gobject-introspection
+}:
+crystal.buildCrystalPackage rec {
+  pname = "gi-crystal";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "hugopl";
+    repo = pname;
```

```suggestion
    repo = "gi-crystal";
```

See: https://github.com/nix-community/nixpkgs-lint/issues/21